### ISSUE TITLE
Simulationcraft Input "raw input" is not in the 1.8.9 format

### ISSUE THREAD

**Issue by herener** [seriallos/raidbots-issues#45]
If you try to copy the raw input from a finished report as an input for a new simulation it will give and error.

"Your SimC addon appears to be out-of-date or invalid - make sure you have version 1.8.5 or higher."

**Comment by seriallos** [comment]
Yeah, this is annoying but is actually intended. I hope to remove it the warning in this scenario at some point in the future.

You can still submit when the warning is there, it's mostly a way to get the attention of folks running older versions of the addon that don't include the version number in the output (which was only added in the last few weeks).

Once the newer version of the addon has a bit more adoption, I'll only show this warning under more specific circumstances that won't affect copy/paste from simc reports.
